src/clientsettingsdlg.cpp

```diff
@@ -42,6 +42,17 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
     layout()->setMenuBar ( pMenu );
 #endif
 
+#if defined( Q_OS_ANDROID ) || defined( ANDROID )
+    // Android too
+    QMenuBar* pMenu      = new QMenuBar ( this );
+    QMenu*    pCloseMenu = new QMenu ( tr ( "Close" ), this );
```

Comment:
Should be `&Close` here and below?

Reply:
Why is the & before C if we use ctrl + w?

Reply:
Because Ctrl W is commonly known for the "Close Window" action.

Reply:
Ok. So we‘ll leave it like that?